.github/workflows/md-link-checker.yml

```diff
@@ -1,5 +1,6 @@
 name: Check Markdown links
 on: 
+  workflow_dispatch:
```

Comment:
This just allows you to manually run the check action. Why remove?

Reply:
It was added?

Reply:
I guess I developed daltonism this morning :D

Thanks for adding it!